Examples/Lenses/AutoLenses.stencil

```diff
@@ -0,0 +1,38 @@
+infix operator *~: MultiplicationPrecedence
```

Comment:
if you'll have time PR after this is merged would be great :) more people we can get using Sourcery the better tool will be